### PR TITLE
Combine backend and module

### PR DESCRIPTION
**Pull request by afbenedict**
Parse json from module to return correct results to client
inlcude print statements in log file for debugging

### CODE REVIEW

File: src/brutus-api/brutus_api/tasks.py [hunk 1]
```diff
@@ -3,8 +3,8 @@
 
 from brutus_api import app
 from brutus_api import nlp
-import json
 import requests
+import json
```
Comment (CtrlC-Root) [inline src/brutus-api/brutus_api/tasks.py]: We should probably use `from flask import json` to be consistent everywhere.

**Comment by CtrlC-Root** [comment]
Minor import issue, otherwise looks good.

**Comment by CtrlC-Root** [comment]
@afbenedict So I think I've fixed the unit tests for the project. After you make the changes above and merge this pull request they should pass on the `master` branch.

**Comment by CtrlC-Root** [comment]
You can see the actual mismatch happening here: https://travis-ci.org/RohitSat/cse5914#L371

**Comment by CtrlC-Root** [comment]
@sinasabet81 @afbenedict alternatively you could merge `master` back into this branch and they should pass in this pull request.

**Comment by sinasabet81** [comment]
@CtrlC-Root so we just need to rebase this to master to get the tests passing?

**Comment by CtrlC-Root** [comment]
You could rebase this branch onto `master` or you could merge `master` into this branch. Either way the tests *should* pass afterwards.

**Comment by CtrlC-Root** [comment]
Or we could merge the pull request ;).

**Comment by sinasabet81** [comment]
💡

**Pull request merged**